tests/test_numeric.py

```diff
@@ -1,4 +1,48 @@
+from decimal import Decimal
+
+import pytest
+
 import trafaret as t
+from trafaret.codes import INVALID_DECIMAL
+
+
+class TestToDecimal:
+    @pytest.mark.parametrize('value, expected', [
+        (0, Decimal('0.0000')),
+        (1000, Decimal('1000.0000')),
+        (1000.0, Decimal('1000.0000')),
+        ('1000', Decimal('1000.0000')),
+        ('1000.0', Decimal('1000.0000')),
+        (-1000, Decimal('-1000.0000')),
+        (-1000.0, Decimal('-1000.0000')),
+        ('-1000', Decimal('-1000.0000')),
+        ('-1000.0', Decimal('-1000.0000')),
+    ])
+    def test_to_decimal(self, value, expected):
+        result = t.ToDecimal().check(value)
+        assert result == expected
+
+    def test_error_code(self):
+        with pytest.raises(t.DataError) as err:
+            t.ToDecimal().check('')
+        assert err.value.code == INVALID_DECIMAL
+
+    def test_extract_error(self):
+        result = t.extract_error(t.ToDecimal(), '')
+        assert result == 'value can\'t be converted to Decimal'
+
+    def test_none_to_decimal(self):
+        with pytest.raises(TypeError) as err:
+            t.extract_error(t.ToDecimal(), None)
+            assert err == 'conversion from NoneType to Decimal is not supported'
+
+    @pytest.mark.parametrize('value, expected', [
+        (t.ToDecimal(), '<ToDecimal>'),
+        (t.ToDecimal[1:], '<ToDecimal(gte=1)>'),
```

Comment:
we forgot describe in documentation this way to set range validation in numerical data types. I added issue https://github.com/Deepwalker/trafaret/issues/82